Source/OmniXaml.Wpf/WpfXamlLoader.cs

```diff
@@ -20,5 +20,15 @@ public object Load(Stream stream, object rootInstance)
         {
             return innerLoader.Load(stream, rootInstance);
         }
+
+        public T Load<T>(Stream stream) where T : class
+        {
+            return innerLoader.Load<T>(stream);
+        }
+
+        public T Load<T>(Stream stream, object instance) where T : class
```

Comment:
i don't know if this makes sense. you're passing in an instance, so what does it return? the same instance? if so shouldn't they be of the same type?